src/bootstrap/src/bin/rustdoc.rs

```diff
@@ -70,7 +70,9 @@ fn main() {
         cmd.arg("--cfg=bootstrap");
     }
     cmd.arg("-Zunstable-options");
+    // #[cfg(bootstrap)]
     cmd.arg("--check-cfg=values(bootstrap)");
```

Comment:
I wonder if this should be also gated for local rebuilds, but I don't think we can even set a custom rustdoc in `config.toml`, so it probably doesn't matter.